### ISSUE TITLE
[BUG] Analyzing tackle-testapp produces no issues

### ISSUE THREAD

**Issue by rromannissen** [konveyor/analyzer-lsp#349]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Konveyor version

0.3-alpha5

### Priority

Blocker

### Current Behavior

No issues are identified in the application:
![Screenshot from 2023-09-29 11-40-53](https://github.com/konveyor/analyzer-lsp/assets/15966456/7d8c5de4-d837-4514-b734-92bc7f59d915)
![Screenshot from 2023-09-29 11-41-04](https://github.com/konveyor/analyzer-lsp/assets/15966456/72a750fc-4261-4e31-9706-f40b14165906)
![Screenshot from 2023-09-29 11-41-12](https://github.com/konveyor/analyzer-lsp/assets/15966456/4913bf0b-1b78-4e9e-8a92-9475c207d433)
![Screenshot from 2023-09-29 11-41-23](https://github.com/konveyor/analyzer-lsp/assets/15966456/7abfef86-db2f-42ae-9646-3348b1b2b68d)

### Expected Behavior

The following issues should have been identified:

- Hard-coded IP address (at src/main/resources/persistence.properties)
- Legacy configuration with io.konveyor.demo.config.ApplicationConfiguration (at io.konveyor.demo.ordermanagement.config.PersistenceConfig via [the custom rule](https://github.com/rromannissen/rhoar-microservices-demo/blob/konveyor/customrules/corporate-framework-config.windup.xml)
- File system - Java IO (in the config-utils dependency at io.konveyor.demo.config.ApplicationConfiguration)

### How Reproducible

Always (Default)

### Steps To Reproduce

1. Analyze tackle-test app with the configuration available in the Environment section.
2. Go to the issues view

### Environment

```markdown
- Fedora 36
- minikube v1.31.2 (--driver=kvm2 --memory=10g --cpus=3)
- Konveyor 0.3 alpha-5
- [tackle-testapp](https://github.com/konveyor/tackle-testapp) application
- Analysis configuration:
  - Source + Dependencies
  - cloud-readiness, linux
  - Application and internal dependencies only
  - Custom rule manually uploaded
  - No additional advanced options
```
